### PR TITLE
No jquery

### PR DESCRIPTION
**Pull request by Haroenv**

### CODE REVIEW

**Comment by LukyVj** [comment]
:shipit: :rocket:

File: index.html [hunk 1]
```diff
@@ -54,5 +54,5 @@ <h3 class="text-muted">Project name</h3>
       </div>
 
     </div> <!-- /container -->
-<script src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
+<!-- <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script> -->
```
Comment (mischah) [inline index.html]: It’s still there.Just with comments. Get that shit outta here!!!!!111one

Comment (Haroenv) [inline index.html]: I wasn't so sure if bootstrap would even work  without jquery, but it seems that it'll luckily work if that bootstrap.js is gone. So poof it all away for me :champagne:

Comment (mischah) [inline index.html]: :cocktail:

**Pull request merged**